pom.xml

```diff
@@ -156,6 +156,13 @@
             <artifactId>h2</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk18on</artifactId>
```

Comment:
This already should be a transitive dependency from `powerauth-java-crypto` (to avoid version problems).